src/ItemEdit.js

```diff
@@ -139,9 +139,17 @@ const ItemEdit = ({ index, roadmap, onChange, onDone }) => {
 
   const handleSuggestionSelect = (index, event) => {
     const values = [...dateFields];
-    if (event.target.name && event.target.name.includes('Stage')) {
+    if (
+      event.target.innerText &&
```

Comment:
Any reason we cannot use `event.target.value` instead of `event.target.innerText`? Something doesn't look like typical grommet with "innerText".

Reply:
We could just do `event.suggestion`:
```
event.suggestion &&
      roadmap.labels.map((label) => label.name).includes(event.suggestion)
```
There is no `event.target.value`

Reply:
Why did we remove the check for the name being 'Stage', to align with the property we set `.stage`?

Reply:
This is no `event.target.name` anymore. I assumed this was something that must have been changed on React's end since it was on the Synthetic Event. If it was something that grommet was adding to an event, then there may be a bug to be fixed.